packages/allow-scripts/src/whichbin.js

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env node
+
+// @ts-check
+const path = require('path')
+const { getOptionsForBin } = require('./index.js')
+
+start().catch((err) => {
+  console.error(err)
+  process.exit(1)
+})
+
+async function start () {
+  const binName = path.basename(process.argv[1])
+  const rootDir = process.cwd()
```

Comment:
Assuming the env var from https://github.com/LavaMoat/LavaMoat/blob/1fbcc2e49979ae5269f9053129da368fb3e90dbe/packages/allow-scripts/src/setup.js#L16 is intended to be used for project selection
```suggestion
  const rootDir = process.env.INIT_CWD || process.cwd()
```